### PR TITLE
Fix missing /dev/stderr for cat2()

### PR DESCRIPTION
**Pull request by pinpox**
Credits to: https://cgit.krebsco.de/stockholm/tree/krebs/5pkgs/simple/irc-announce/default.nix?id=1e915baafbc56c4c626a3c3d042ac0e7d6276561

### CODE REVIEW

File: pkgs/irc-announce/default.nix [hunk 1]
```diff
@@ -32,7 +32,12 @@
   # echo2 and cat2 are used output to both, stdout and stderr
   # This is used to see what we send to the irc server. (debug output)
   echo2() { echo "$*"; echo "$*" >&2; }
-  cat2() { tee /dev/stderr; }
+  cat2() {
+    awk '{
+      print $0
+      print $0 > "/dev/stderr"
+    }'
```
Comment (Mic92) [inline pkgs/irc-announce/default.nix:39]:
```suggestion
    tee /dev/stderr /dev/stdout;
```

File: pkgs/irc-announce/default.nix [hunk 2]
```diff
@@ -32,7 +32,12 @@
   # echo2 and cat2 are used output to both, stdout and stderr
   # This is used to see what we send to the irc server. (debug output)
   echo2() { echo "$*"; echo "$*" >&2; }
-  cat2() { tee /dev/stderr; }
```
Comment (Mic92) [inline pkgs/irc-announce/default.nix:35]: The code already does this.
See `tee(1)`:

```
DESCRIPTION
       Copy standard input to each FILE, and also to standard output.
```

Comment (Lassulus) [reply pkgs/irc-announce/default.nix:35]: /dev/stderr is not a file in a systemd service

Comment (4z3) [reply pkgs/irc-announce/default.nix:35]: References:
- https://cgit.krebsco.de/stockholm/commit/krebs/5pkgs/simple/irc-announce/default.nix?id=5281ea110297a40707a4b94855d5d3da04c71e37
- https://github.com/poettering/systemd/commit/c9cc1a2d0c591e571f0e27dff1e9c03d341a0d7d

**Pull request merged**